CommonLibSF/include/SFSE/Interfaces.h

```diff
@@ -107,15 +107,11 @@ namespace SFSE
 
 		constexpr void PluginVersion(std::uint32_t a_version) noexcept { pluginVersion = a_version; }
```

Comment:
forgot to do this--this should accept a `REL::Version` and use `a_version.pack()` in order to play nice with the `target_commonlibsf_properties` command in CommonLibSF.cmake

Reply:
Done, I will say though idk how much we should let existing plugins inform our decisions for now, it's still early days--not like changing something would break hundreds of plugins